internal/app/go-filecoin/connectors/retrieval_market/provider_test.go

```diff
@@ -49,7 +50,12 @@ func TestRetrievalProviderConnector_UnsealSector(t *testing.T) {
 	tf.UnitTest(t)
 	ctx := context.Background()
 	sectorID := rand.Uint64()
-	fixtureFile := "../../../../../fixtures/constants.go"
```

Comment:
This blew my mind. I think I've captured the intent of the test without referring to this file now.